services/cards-publication/src/main/java/org/opfab/cards/publication/services/CardProcessingService.java

```diff
@@ -154,7 +156,8 @@ private void processOneCard(Card card, Optional<CurrentUserWithPerimeters> user,
                     card.getProcess(), card.getState()));
             }
 
-            if (!cardPermissionControlService.isCardPublisherInUserEntities(card, user.get()))
+            if (!cardPermissionControlService.isCardPublisherInUserEntities(card, user.get()) &&
```

Comment:
This should be tested with the publisher type : 

if publisher is USER and is not the user himself --> Error 
if publisher is ENTITY and is not in userEntity --> Error

Add unit test

Reply:
Done.